src/utils/walletSearch.js

```diff
@@ -40,9 +40,11 @@ export const wallet = {
     const handleWallets = async function ({ wallets, newWallet }) {
       detector.stopScan();
       const connected = await this.client.connectToWallet(await newWallet.getConnection());
-      this.client.selectNode(connected.networkId); // connected.networkId needs to be defined as node in RpcAepp
+      // TODO re-enable once fixed
```

Comment:
What's not working with this, was working well in my testing

Reply:
I think we should only target most up-to-date sdk anyways, let's ask the superhero wallet to be updated